guides/common/modules/proc_registering-a-host.adoc

```diff
@@ -7,13 +7,13 @@ You can register a host by using registration templates and set up various integ
 * Your user account has a role assigned that has the `create_hosts` permission.
 * You must have root privileges on the host that you want to register.
 ifdef::satellite,orcharhino[]
-* {ProjectServer}, any {SmartProxyServers}, and your host must be synchronized with the same NTP server, and have a time synchronization tool enabled and running.
-* An activation key must be available for your host.
+* {ProjectServer}, any {SmartProxyServers}, and all hosts must be synchronized with the same NTP server, and have a time synchronization tool enabled and running.
```

Comment:
I recommend rewriting this prerequisite so that it is active voice. "Ensure that ...." or "Synchronize ... with the same NTP server." 

"time synchronization tool enabled and running" looks like a separate prerequisite.